src/main/java/org/synyx/urlaubsverwaltung/sickdays/web/SickDaysOverviewViewController.java

```diff
@@ -72,7 +73,7 @@ public String filterSickNotes(@ModelAttribute("period") FilterPeriod period) {
     public String periodsSickNotes(@RequestParam(value = "from", required = false) String from,
                                    @RequestParam(value = "to", required = false) String to, Model model) {
 
-        FilterPeriod period = new FilterPeriod(Optional.ofNullable(from), Optional.ofNullable(to));
```

Comment:
Hiermit brechen wir die API und sollten zum einen einen Test schreiben aber auch schauen ob wir sie auch brechen wollen und das ganze dann in 4.0 machen